Framework/Reflectometry/test/ReflectometryReductionOneAuto2Test.h

```diff
@@ -8,7 +8,7 @@
 
 #include <cxxtest/TestSuite.h>
 
-#include "MantidAlgorithms/ReflectometryReductionOneAuto2.h"
+#include "MantidReflectometry/ReflectometryReductionOneAuto2.h"
```

Comment:
Is the version 2 of this algorithm still needed?

Reply:
Yes - some properties were removed in v3 and we cannot replay history of old workspaces that used them without it